source/adapters/level_zero/platform.cpp

```diff
@@ -304,14 +304,30 @@ ur_result_t ur_platform_handle_t_::populateDeviceCacheIfNeeded() {
 
   uint32_t ZeDeviceCount = 0;
   ZE2UR_CALL(zeDeviceGet, (ZeDriver, &ZeDeviceCount, nullptr));
+  uint32_t ZesDeviceCount = 0;
+  if (ZesDriver) {
+    ZE2UR_CALL(zesDeviceGet, (ZesDriver, &ZesDeviceCount, nullptr));
```

Comment:
we might again need version check before calling zesDeviceGet to make sure we have support?